docker/etc/templates/app.ini

```diff
@@ -1,16 +1,23 @@
 [repository]
 ROOT = /data/git/repositories
 
+[repository.local]
+LOCAL_COPY_PATH = /data/gitea/tmp/local-repo
+
 [repository.upload]
 TEMP_PATH = /data/gitea/uploads
 
 [server]
 APP_DATA_PATH = /data/gitea
+LFS_CONTENT_PATH = /data/gitea/data/lfs
```

Comment:
Yes, you are right. They are all `git` related stuff, so it's better they stay together. I'll change it to `/data/git/lfs` as you suggested.